### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by visahak** [IBM/FfDL#56]
I am getting this error while running `make docker-build` command on a mac environment.

```(cd etc/dlaas-service-base; make build)
cd build/grpc-health-checker && make install-deps build-x86-64
glide -q install
[WARN]	The name listed in the config file (github.ibm.com/deep-learning-platform/grpc-health-checker) does not match the current location (github.com/IBM/FfDL/etc/dlaas-service-base/build/grpc-health-checker)
rm -rf bin/
CGO_ENABLED=0 GOOS=linux go build -ldflags "-s" -a -installsuffix cgo -o bin/grpc-health-checker
docker build -q -f Dockerfile.ubuntu -t dlaas-service-base:ubuntu16.04 .
sha256:6975b033728017afc0f9a2dd9978e76331411d323fef07c9b8600959cccdbf4a
docker tag dlaas-service-base:ubuntu16.04 docker.io/ffdl/dlaas-service-base:ubuntu16.04
docker build -q -f Dockerfile.alpine -t dlaas-service-base:alpine3.3 .
sha256:9e367f913da06d9758a26936bff18c23587e5eebfd424f9793a0cd488742e75b
docker tag dlaas-service-base:alpine3.3 docker.io/ffdl/dlaas-service-base:alpine3.3
(full_img_name=ffdl-metrics; \
		cd ./metrics/ && (if [ "minikube" = "minikube" ]; then eval $(minikube docker-env); fi; \
			docker build -q -t docker.io/ffdl/$full_img_name .))
Sending build context to Docker daemon  11.26MB
Step 1/6 : FROM dlaas-service-base:ubuntu16.04
pull access denied for dlaas-service-base, repository does not exist or may require 'docker login'
make[1]: *** [.docker-build] Error 1
make: *** [docker-build-metrics] Error 2```

**Comment by Tomcli** [comment]
It looks like you are using "minikube" as your build environment. If you want to build and run FfDL on your Minikube's Docker daemon. Then you need to run `eval $(minikube docker-env)` to point your Docker CLI to the Minikube's Docker daemon. 

If you just want to build FfDL on your default Docker daemon, run `export VM_TYPE=none` to avoid the makefile building FfDL docker images with your Minikube's Docker daemon.

**Comment by visahak** [comment]
Sounds good. Modifying the developer guide to state that would be useful.

**Comment by Tomcli** [comment]
Thanks @visahak, we are still in early stage, so we will have a lot of documentation changes coming soon.

**Comment by animeshsingh** [comment]
Thanks @visahak - please also submit a PR if you think mentioning this at a certain point in flow would be better.

**Comment by lingarajonline** [comment]
I am getting error with one pod that gets into CrashLoopBackoff status. I have tried a few times with clean 'helm install' but get the same error.

ffdl-trainingdata-86c5578b75-v884m   0/1       CrashLoopBackOff   5          7m

Any suggestion, how to get past this error and have a clean deployment of this fabric? Thanks

**Comment by lingarajonline** [comment]
<img width="1663" alt="screen shot 2018-04-09 at 6 30 42 pm" src="https://user-images.githubusercontent.com/34495576/38531339-36bced2a-3c24-11e8-91cc-6b7c5532412d.png">

**Comment by animeshsingh** [comment]
@visahak can this be closed? Still running in build issues?

**Comment by Tomcli** [comment]
Resolved in #73
